src/module.c

```diff
@@ -945,7 +953,7 @@ JL_DLLEXPORT jl_value_t *jl_module_names(jl_module_t *m, int all, int imported)
             break;
         jl_sym_t *asname = b->globalref->name;
         int hidden = jl_symbol_name(asname)[0]=='#';
-        if ((b->exportp ||
+        if (((qualified ? b->publicp : b->exportp) ||
```

Comment:
```suggestion
        if ((b->exportp ||
             (qualified && b->publicp) ||
```

Is this equivalent? Seems clearer if so that `qualified` only tests the `publicp` bit.

Reply:
Should be